iroh-bytes/src/protocol.rs

```diff
@@ -31,13 +33,77 @@ impl Handshake {
     }
```

Comment:
This PR originally bumped the protocol version from 2 -> 3. I've reverted the change, keeping the version at 2, because we haven't cut a release with version 2 as the identifier. Others feel free to disagree